webpack.config.js

```diff
@@ -131,11 +134,14 @@ module.exports = (env, argv) => {
             }
         },
         plugins: [
-            new WebpackDashDynamicImport(),
+            new MiniCssExtractPlugin({
+                filename: `${dashLibraryName}.css`,
+            }),
+            //new WebpackDashDynamicImport(),
```

Comment:
Had to comment this one out in order to be able to run `npm run build`, but I guess someone knowing the details of `WebpackDashDynamicImport` would be able to figure out how to include it again.